README.md

```diff
@@ -34,46 +34,118 @@ Simmy offers the following chaos-injection policies:
 
 |Policy| What does the policy do?|
 | ------------- |------------- |
-|**[Fault](#Inject-fault)**|Injects exceptions or substitute results, to fake faults in your system.|
+|**[Exception](#Inject-exception)**|Injects exceptions in your system.|
+|**[Result](#Inject-result)**|Substitute results to fake faults in your system.|
 |**[Latency](#Inject-latency)**|Injects latency into executions before the calls are made.|
 |**[Behavior](#Inject-behavior)**|Allows you to inject _any_ extra behaviour, before a call is placed. |
 
 # Usage
 
-## Inject fault
+## Inject exception
 ```csharp
-var chaosPolicy = MonkeyPolicy.InjectFault(
-    Exception | Func<Context, CancellationToken, Exception> fault,
-    double | Func<Context, CancellationToken, double> injectionRate, 
-    Func<bool> | Func<Context, CancellationToken, bool> enabled 
-);
+var chaosPolicy = MonkeyPolicy.InjectException(Action<InjectOutcomeOptions<Exception>>);
+```
+
```

Comment:
Wondering if we should give an example here as well as the 'abstract' syntax? Maybe something like below:

((( Maybe we will end up moving all the examples from **Basic examples Step 1 Set up the Monkey Policy**. Maybe we could just merge that whole section into here - what do you think? )))

    var chaosPolicy = MonkeyPolicy.InjectException(Action<InjectOutcomeOptions<Exception>>);

> For example

    var chaosPolicy = MonkeyPolicy.InjectException(with =>
        with.Fault(new SocketException(errorCode: 10013))
            .InjectionRate(0.05)
            .Enabled()
        );